### ISSUE TITLE
class is missing in the ul tags

### ISSUE THREAD

**Issue by basd82** [joomla/joomla-cms#8945]
#### Steps to reproduce the issue
Currently the tag is like this, `<ul>`

#### Expected result
It should be `<ul class="tagspopular"`>

#### solution
`on line 18 in joomla-cms/modules/mod_tags_popular/tmpl/default.php`

`change <ul> in <ul class="tagspopular<?php echo $moduleclass_sfx; ?>">`

**Comment by basd82** [comment]
missing sollution code for line 18 is :
<ul class="tagspopular<?php echo $moduleclass_sfx; ?>">

**Comment by bertmert** [comment]
see line 14:
```
<div class="tagspopular<?php echo $moduleclass_sfx; ?>">
```
I don't understand why ul should get same classes. Just makes it mor complicated in CSS.

Instead use something like
```
div.tagspopular ul{
 ....
}
```

**Comment by bertmert** [comment]
Or use a template override for default.php

**Comment by brianteeman** [comment]
@basd82 is correct from a consistency perspective - if you check the other modules that produce lists then they work the way he expects

**Comment by basd82** [comment]
In all other modules it’s done this way (as far as I think the correct way), but in mod_tags_popular and mod_tags_similar not.

So I suggest keep to the standard.


Van: bertmert [mailto:notifications@github.com]
Verzonden: woensdag 20 januari 2016 11:43
Aan: joomla/joomla-cms <joomla-cms@noreply.github.com>
CC: Bas van den Dikkenberg <bas@Dikkenberg.net>
Onderwerp: Re: [joomla-cms] class is missing in the ul tags (#8945)


Or use a template override for default.php

**Comment by basd82** [comment]
If you do this way, you can easily change the layout of a singel module<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8945">issues.joomla.org/joomla-cms/8945</a>.</sub>

**Comment by bertmert** [comment]
I can live with this change but have to fix 2 sites (not a problem).

> If you do this way, you can easily change the layout of a singel module
I think It's easier to change if you have a surrounding div with this class ;-)

I think the surrounding div should also be removed then.
Why: Use a layout style html5 with your module change and you'll see that we have now 3 times the same $moduleclass_sfx and 2 times "tagspopular". One for div, one for ul. You have to force CSS with very "hard" rules. Leading div, ul and so on.

> is correct from a consistency perspective

Yes.

**Comment by flash1452** [comment]
@brieteeman I would like to work on this.

**Comment by brianteeman** [comment]
@flash1452 you dont need anyones permission<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8945">issues.joomla.org/joomla-cms/8945</a>.</sub>

**Comment by brianteeman** [comment]
Closed as we have a PR for testing #9388<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8945">issues.joomla.org/joomla-cms/8945</a>.</sub>
